sources/effects.c

```diff
@@ -85,7 +85,7 @@ int write_oriented_tap_sequence(int fd, int numLocations, ...) {
     if (action == SLEEP) {
       usleep(50000);
     } else if (action == LONG_SLEEP) {
-      usleep(200000);
+      usleep(400000);
```

Comment:
We already discussed making this configurable, but since this is quite a big bump perhaps we could also somehow make this dependent on whether the users wants to use the long hold feature?

Reply:
Yeah, sorry I think I let this slip into the PR by accident. Hopefully this issue won't be relevant for too much longer, though, if we're able to somehow hook into rm-hacks (I recently opened mb1986/rm-hacks#140, we'll see how that goes).

Reply:
`rm-hacks` looks quite cool in terms of what it can do, but honestly I'm not a huge fan of installing random binaries on my machines